Formula/brainfuck.rb

```diff
@@ -0,0 +1,30 @@
+class Brainfuck < Formula
+  desc "Interpreter for brainfuck written in C"
+  homepage "https://github.com/fabianishere/brainfuck"
+  url "https://github.com/fabianishere/brainfuck/archive/2.7.1.tar.gz"
+  sha256 "06534de715dbc614f08407000c2ec6d497770069a2d7c84defd421b137313d71"
+
+  head "https://github.com/fabianishere/brainfuck.git"
+
+  option "with-debug", "Compile interpreter with debug support"
```

Comment:
This option should probably be removed as debug does not work well with superenv.

Reply:
Why would that be the case? (I'm not really that familair with superenv, but this option seems quite harmless to me) I could try to make the description of this option more clear, but what it does currently is compile the binary with a command which is not an official part of the brainfuck language. This command display useful (debugging) information about the state of the brainfuck program.

Reply:
Ah ok. So it's not like `-g`. Is there any downside to always enabling this?

Reply:
Yes, it doesn't conform to the language specification. This is not a big downside, but 1. adding non-standard features to well defined languages feels weird (i'm not sure if it is comparable, but imagine if you would do this for C) 2. brainfuck was originally meant to be as small as a compiler as possible, so any bloat should be avoided imo.